vulcanai/models/basenetwork.py

```diff
@@ -843,31 +898,20 @@ def forward(self, inputs, **kwargs):
         return self.network(output)
 
     @torch.no_grad()
-    def forward_pass(self, data_loader, transform_outputs=False,
+    def forward_pass(self, data_loader,
                      transform_callable=None, **kwargs):
         """
         Allow the user to pass data through the network.
-
         Parameters:
             data_loader : DataLoader
                 DataLoader object to make the pass with.
-            transform_outputs : boolean
-                If true, transform outputs using metrics.transform_outputs.
-                If no transform_callable is provided then the defaults in
-                metrics.transform_outputs will be used: class converstion for
-                one-hot encoded, and identity for one-dimensional outputs.
-                Multiple class multiple outputs are not yet supported.
             transform_callable: callable
-                Used to transform values if transform_outputs is true,
-                otherwise defaults in metrics.transform_outputs will be used.
-                An example could be np.round
+                A torch function. e.g. torch.round()
```

Comment:
does it have to be a torch function? I see some juggling with numpy matrices

Reply:
mmmmm... no that all should be removed. it's going through to forward_pass